android/build.gradle

```diff
@@ -27,7 +27,7 @@ allprojects {
 }
 
 ext {
-    compileSdkVersion = 31
+    compileSdkVersion = 33
```

Comment:
reasoning behind min complile sdk version update?

Reply:
Test cases was not working for 31 so upgraded the compilesdkversion 33